app/clients/zendesk_sell_client.py

```diff
@@ -0,0 +1,94 @@
+import json
```

Comment:
Don't think you need the `_client.py` suffix for this class, how are other classed named?

Reply:
This was inline with the notification-util. it will be removed in the upcoming change